apis/nucleus/src/components/sheetGlue.jsx

```diff
@@ -14,10 +23,12 @@ export default function glue({ halo, element, model, initialSnOptions, initialSn
       initialSnPlugins={initialSnPlugins}
       initialError={initialError}
       onMount={onMount}
+      navigation={navigation}
     />,
     element,
     model.id
   );
+  navigation.setSheetRef(sheetRef);
 
   const unmount = () => {
     root.remove(portal);
```

Comment:
Below here is a `model.on('closed', unmount)`, won't that potentially mess with things?